c2cgeoportal/scaffolds/create/vars_+package+.yaml_tmpl

```diff
@@ -115,6 +115,10 @@ vars:
             The GeoMapfish team
         smtp_server: smtp.example.com
 
+    headers:
+        mapserver:
+            cors: *
```

Comment:
Use from the Cartoriviera portal with the WMSBrowser the WMS from the Vaud portal without the OGSProxy...